frontend/src/components/regions/CardDetails/Review.js

```diff
@@ -58,6 +64,18 @@ const Review = ({ review }) => {
         title={
           <Typography className={classes.timeFont}>
             {timestamp.toLocaleString()}
+            <Button
```

Comment:
Can a change in the icon color make this more user friendly and engaging? Please see screenshot. On a UI perspective, the icon doesn't look good near the review text and clunks things up.

Reply:
It still doesn't look dead obvious. We need it to be so abvious that a brick could understand what the button was for. 
If we can't make it obvious then we have 2 options:

1. get rid of the trimming so that if folks dont notice the button they can still get their work done
2. acept that we are just going to need the humans on our team to waste time explaining the same thing over and over again 

Both of these seem pretty bad to me.  How about using words to make the button clear and meaningful? Eg: [show full review] or something.  I would rather have something ugly that works, than something beautiful that trips people up.